### ISSUE TITLE
Discover feed options, like in Chrome

### ISSUE THREAD

**Issue by Lolzen** [kiwibrowser/src.next#523]
**Is your feature request related to a problem? Please describe.**
The Discover feed on Chrome seems to be more "personizable", in the sense of you can choose to opt out of chosen news outlets right from the feed options (see screenshots)

**Describe the solution you'd like**
I'd like to be able to personalize my discover feed like in Chrome

**Describe alternatives you've considered**
Using Chrome for the news feed, Kiwi for everything else.

**Additional context**
Discover feed on Chrome
![Screenshot_20220222-112122547](https://user-images.githubusercontent.com/327588/155115237-87abb367-5abd-49c6-ad71-aac565ffef7a.png)
Discover feed options on Chrome
![Screenshot_20220222-112130091](https://user-images.githubusercontent.com/327588/155115365-fd02780e-50de-4736-8019-6c5667d7c14c.png)
Discover feed on Kiwi (lacks these buttons)
![Screenshot_20220222-112139705_edit](https://user-images.githubusercontent.com/327588/155115423-c3adaebd-57be-48ea-ac10-d5d4f2e99b5e.png)

I don't know if it is possible to achieve, but i haven't seen a similar request/report on github so i'm opening this one.

**Comment by gnadelwartz** [comment]
Nice suggestion, I also don't like the Boulevardblatt Bildzeitung as news source :) Arnaud is working on customisation, may be can implement something like this

**Comment by kiwibrowser** [comment]
Yes I have a problem with the news too, they are too gloomy, for me it's "war", "covid", "war", "covid", so we need to find new things and a way how to personalise them better.
What I'm going to do, is open the code (on this Friday) for the JS that controls the feed so we can starts customising it together, because there is not a single view.
I just have a technical issue that I need to sort out, is how to make it safe *and* remotely updatable. Because if we want to make cool things, then we need to have access to local history, but if we want to make it remotely updatable, then somewhere it has to be remotely hosted, then there is a risk.
So I wonder if there is a way to hedge against this risk, and still offer the personalisation. (if you have any opinion, you are very welcome as well).

The current code can be seen at view-source:chrome-search://local-ntp/bridge.html if you are curious.
It's a JS code controlling the native view.

**Comment by 424344** [comment]
I'd like some news customization: I'm not interested in sports so I'd like to hide sports news completely.
